sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusModelFactory.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Messaging.ServiceBus.Primitives
+{
+    /// <summary>
+    /// This class contains methods to create certain ServiceBus models.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public static class ServiceBusModelFactory
+    {
+        /// <summary>
+        /// Creates a new ServiceBusReceivedMessage instance for mocking.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static ServiceBusReceivedMessage ServiceBusReceivedMessage(
+            ServiceBusMessage sentMessage,
```

Comment:
This is internal so I don't think we need to expose it. Instead, we can expose each of the public properties as parameters.

Reply:
Exposed the public properties as parameters and create the message in the method.